README.md

```diff
@@ -48,6 +48,24 @@ console.log('Hello remark-code-import!');
 
 The file path is relative to the markdown file path.
 
+You may also specify specific lines or ranges:
+
+````md
+```js file=./say-hi.js#L3
+```
+
+```js file=./say-hi.js#L3-L6
+```
+
+```js file=./say-hi.js#L3-
+```
+
+```js file=./say-hi.js#-L6
```

Comment:
I don't think this is useful as it's the same as `#L1-L6` and it complicates the rule a little bit. I believe we should remove it, WDYT?

Reply:
Indeed. I removed this and added a test case to check for it: https://github.com/o1iver/remark-code-import/blob/0051e274e9340b0be4b9db5e78641bc243e165d7/test.js#L65.